score/application.py

```diff
@@ -0,0 +1,56 @@
+from config import settings
+import click
+from pathlib import Path
+import logging
+
+from etl.sources.census.etl_utils import reset_data_directories as census_reset
+from etl.sources.ejscreen.etl_utils import reset_data_directories as ejscreen_reset
+from utils import remove_files_from_dir
+from etl.sources.census.etl import download_census_csvs
+
+settings.APP_ROOT = Path.cwd()
+
+
+@click.group()
+def cli():
```

Comment:
Could we add docstrings to all our functions? It'll help out a lot in the long run!

Reply:
Tracking on this ticket: https://app.zenhub.com/workspaces/justice40-60993f6e05473d0010ec44e3/issues/usds/justice40-tool/255